src/app/week-log/week-log.component.spec.ts

```diff
@@ -46,14 +46,12 @@ describe('WeekLogComponent', () => {
     expect(component).toBeTruthy();
   });
 
-  it('should calculate totalSum', () => {
-    expect(component.totalSum).toBe(dayLog1.hoursLogged + dayLog2.hoursLogged);
-  });
-
-  it('should display totalSum', () => {
+  it('should calculate correctly total sum of logged hours and display it', () => {
+    const totalSum = dayLogs.reduce((sum, val) => sum + val.hoursLogged, 0);
```

Comment:
It is handy to prefix expected values, e.g. with 'expected' in order to distinguish values obtained and expected values.